synapse/models/inet.py

```diff
@@ -3592,6 +3602,23 @@ def getModelDefs(self):
                             'doc': 'Current profile details associated with the account.'}),
                     )),
 
+                    ('inet:service:follower:type:taxonomy', {}, ()),
+                    ('inet:service:follower', {}, (
+
+                        ('account', ('inet:service:account', {}), {
+                            'doc': 'The account which initiated the relationship.'}),
+
+                        ('target', ('inet:service:account', {}), {
```

Comment:
Would we still expect "mutual relationships" (like a "connection" on LinkedIn) to use two instances of this form, one for each "direction"? That's one option but it seems like with the addition of the :type prop we could "get away' with one ( I initiated the connection request (:account), you accepted (:target), our :type is "connection" or "mutual" or whatever)?

Reply:
Yep, we still need to keep these directional to account for traversing them correctly. Having mixed source/target use would make pivoting across them very complex. :+1: